storybook/src/core/Breadcrumbs.stories.tsx

```diff
@@ -30,6 +33,14 @@ export default {
   parameters: {
     docs: {
       page: mdx,
+      source: { type: "dynamic", excludeDecorators: true },
+      transformSource: (source) => {
+        const correctedSource = source
+          .replace(/<Breadcrumb>/g, "<Breadcrumbs.Breadcrumb>")
+          .replace(/<\/Breadcrumb>/g, "</Breadcrumbs.Breadcrumb>")
```

Comment:
/Breadcrumb/g, "Breadcrumbs.Breadcrumb" ?

Reply:
If I replace it like that, `<Breadcrumbs...` will also get replaced, so in this case I had to do it like this.